src/index.test.ts

```diff
@@ -1,7 +1,57 @@
 import { describe, it, expect } from 'vitest';
+import { render } from '@testing-library/svelte';
+import { JSDOM } from 'jsdom';
+import NotificationList, {
+  getPositionClasses,
+  Position
+} from './lib/components/NotificationList.svelte';
 
-describe('sum test', () => {
-  it('adds 1 + 2 to equal 3', () => {
-    expect(1 + 2).toBe(3);
+const dom = new JSDOM();
+global.window = dom.window;
+global.document = dom.window.document;
```

Comment:
I think vitest does this for us already 🤔, check out how we do here:
https://github.com/whizzes/quicklink-dashboard/blob/main/test/TextField.test.ts

We just set it up in the `vitest.config.ts` file:

https://github.com/whizzes/quicklink-dashboard/blob/main/vitest.config.ts#L6-L9

Can you try this approach please?